ThunderCloud/LocalisedString.swift

```diff
@@ -44,6 +44,9 @@ public extension String {
     func localised(with key: String, paramDictionary: [String: Any]?) -> String {
         var string = NSAttributedString(string: self).localised(with: key, paramDictionary: paramDictionary).string
         string.localisationKey = key
+        if LocalisationsTool.showDebugLocalisations && !string.hasPrefix("[\(key)]") {
```

Comment:
Also, feel like I'm missing something obvious, but when the prefix check?

Reply:
So basically I originally added this into all 3 base string methods... It turns out they all end up calling the method on localised string... but then I also added the call to `StormLanguageController` itself... If any of that logic changes I thought it best to keep it safe so we don't break this functionality (So we still add the prefix at the call-site even if underlying methods are changed to not add it) at a future date. Happy to remove though if we wish and rely entirely on `StormLanguageController`?

Reply:
IMO it's nice to do it in one place if that's possible, and others reference that. 
But I'm happy to go with what you think is best here!

Reply:
Let's leave it for now, can always clean up at a later time if we reckon this is unlikely to change :)